object.c

```diff
@@ -98,6 +98,15 @@ rb_obj_embedded_size(uint32_t numiv)
     return offsetof(struct RObject, as.ary) + (sizeof(VALUE) * numiv);
 }
 
+/*!
```

Comment:
Unrelated, but this was unclear (what does "hide" mean?) and didn't have any docs, so I just added some.

Reply:
Hide the object from ObjectSpace, as written in `hidden_identity_hash_new` comment.

Reply:
Heh yeah I got it, but I didn’t in first read, which is why I added the doc comment for future readers